testapps/citus_master/citus_router.py

```diff
@@ -2,11 +2,11 @@
 
 from django.conf import settings
 
-citus_masters = [db for db in settings.DATABASES.values() if db.get('ROLE', None) == 'citus_master']
+citus_masters = [name for name, db in settings.DATABASES.items() if db.get('ROLE', None) == 'citus_master']
```

Comment:
nit: I realize this was not added or changed in this commit, but...
```suggestion
citus_masters = [name for name, db in settings.DATABASES.items() if db.get('ROLE') == 'citus_master']
```